src/app/ldtk_entity.rs

```diff
@@ -242,6 +242,43 @@ use crate::app::register_ldtk_objects::RegisterLdtkObjects;
 ///     entity_instance: EntityInstance,
 /// }
 /// ```
+///
+/// ### `#[with(...)]`
+///
+/// Indicates that this component or bundle should be initialized with the given
+/// function.
+///
+/// Note: The given function should have signature `fn (entity: EntityInstance) -> T`
+/// where `T` is the field type. The function should also be accessible in the scope.
+///
+/// ```
+/// use bevy::prelude::*;
+/// use bevy_ecs_ldtk::prelude::*;
+///
+/// #[derive(Component, Default)]
+/// pub struct Player;
+/// #[derive(Component, Default, Clone)]
+/// pub struct Money(f32);
+///
```

Comment:
You can hide the boring stuff from the docs by preceding the lines with `# `:
```suggestion
/// # use bevy::prelude::*;
/// # use bevy_ecs_ldtk::prelude::*;
/// # #[derive(Component, Default)]
/// # pub struct Player;
/// # #[derive(Component, Default, Clone)]
/// # pub struct Money(f32);
```

Reply:
oh! I thought it was an error in the suggestion, that's really cool. Updating now

Reply:
Yeah! in case you don't know you can always view it to make sure it looks good with `cargo doc --open`